model/src/main/scala/uk/gov/ons/addressIndex/model/server/response/address/AddressResponseError.scala

```diff
@@ -221,4 +221,20 @@ object CountryBoostsInvalidError extends AddressResponseError(
 object CountryDeprecationError extends AddressResponseError(
   code = 42,
   message = "The use of fromsource is deprecated and cannot be used in conjuction with eboost, nboost, sboost and wboost."
-)
\ No newline at end of file
+)
+
+object TimeoutNotNumericAddressResponseError extends AddressResponseError(
+  code = 43,
+  message = "Timeout parameter is not numeric"
+)
+
+object TimeoutTooSmallAddressResponseError extends AddressResponseError(
+  code = 44,
+  message = "Timeout parameter is too small, minimum (milliseconds) = 50"
+)
+
+object TimeoutTooLargeAddressResponseError extends AddressResponseError(
+  code = 45,
+  message = "Timeout parameter is too large, maximum (milliseconds) = *"
```

Comment:
Should there be a numerical value here?

Reply:
No. It substitutes the value from the config file.

Reply:
Sweet.